File: GO/logica.go
package main

import (
    "fmt"
)

func ehPar(n int) bool {
    return n%2 == 0
}

func main() {
    var numero int
    fmt.Print("Digite um número: ")
    fmt.Scan(&numero)

    if ehPar(numero) {
        fmt.Println("O número é par.")
    } else {
        fmt.Println("O número é ímpar.")
    }

    fmt.Println("Contando até", numero, ":")
    for i := 1; i <= numero; i++ {
        fmt.Println(i)
    }
}

File: KOTLIN/logica.kt
fun ehPar(n: Int): Boolean {
    return n % 2 == 0
}

fun main() {
    print("Digite um número: ")
    val numero = readln().toInt()

    if (ehPar(numero)) {
        println("O número é par.")
    } else {
        println("O número é ímpar.")
    }

    println("Contando até $numero:")
    for (i in 1..numero) {
        println(i)
    }
}
